net-libs/srt renamed the SRTO_TSBPDDELAY constant to SRTRO_LATENCY.
https://github.com/Haivision/srt/pull/1364
https://bugs.gentoo.org/758062
--- b/modules/access/srt.c
+++ a/modules/access/srt.c
@@ -165,7 +165,7 @@ static bool srt_schedule_reconnect(strea
 
     /* Set latency */
     i_latency = var_InheritInteger( p_stream, "latency" );
-    srt_setsockopt( p_sys->sock, 0, SRTO_TSBPDDELAY,
+    srt_setsockopt( p_sys->sock, 0, SRTO_LATENCY,
         &i_latency, sizeof( int ) );
 
     psz_passphrase = var_InheritString( p_stream, "passphrase" );
--- a/modules/access_output/srt.c
+++ b/modules/access_output/srt.c
@@ -162,7 +162,7 @@ static bool srt_schedule_reconnect(sout_
 
     /* Set latency */
     i_latency = var_InheritInteger( p_access, "latency" );
-    srt_setsockopt( p_sys->sock, 0, SRTO_TSBPDDELAY,
+    srt_setsockopt( p_sys->sock, 0, SRTO_LATENCY,
         &i_latency, sizeof( int ) );
 
     if ( psz_passphrase != NULL && psz_passphrase[0] != '\0')
